### PR TITLE
Pods 5120: adding practioner yml file

### PR DESCRIPTION
**Pull request by Ashwani1403**

### CODE REVIEW

File: conf/services/pension-practitioners.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+serviceName: Managing Pension Schemes
+serviceHeaderName: Managing Pension Schemes
```
Comment (JoPintoPaul) [inline conf/services/pension-practitioners.yml:2]: @Ashwani1403 The parameter `serviceHeaderName` is no longer needed.

File: conf/services/pension-practitioners.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - description: The link focus colour is incorrect in some cards, caused by over-specific style sheet styling. This will be fixed by lowering the specificity of the CSS.
+    date: 2021-03-31
+  - description: Some tables use code that will cause issues with assistive technology. This will be fixed by updating the code used in tables.
+    date: 2021-03-31
+  - description: The spacing between page titles and body text is not always consistent and results in a disjointed user experience. This doesn’t meet WCAG 2.1 success criterion 1.3.1 (A) (Info and Relationships). This will be fixed by using the CSS for styling.
+    date: 2021-03-31
+serviceLastTestedDate: 2020-08-30
+statementVisibility: public
+statementCreatedDate: 2020-09-18
+statementLastUpdatedDate: 2021-02-03
```
Comment (JoPintoPaul) [inline conf/services/pension-practitioners.yml:29]: Please update `statementLastUpdatedDate` to 2021-02-21

File: conf/services/pension-practitioners.yml (hunk 1 above)
Comment (Ashwani1403) [reply conf/services/pension-practitioners.yml:2]: ok, update done

**Review by JoPintoPaul — APPROVED** [review]

**Comment by JoPintoPaul** [comment]
Closing as this has been superseded by PR #356

**Pull request closed without merging**